iOS/DuckDuckGo/DataBrokerProtectionDebugViewController.swift

```diff
@@ -1,276 +0,0 @@
-//
-//  DataBrokerProtectionDebugViewController.swift
```

Comment:
I just moved this file into a `DBP` directory, git didn't handle it that well apparently